fv3core/fv3core/initialization/baroclinic_jablonowski_williamson.py

```diff
@@ -0,0 +1,166 @@
+import math
+
+import numpy as np
+
+import fv3core.utils.global_constants as constants
+from fv3core.grid import great_circle_distance_lon_lat
+
+
+"""
+  Functions for computing components of a baroclinic perturbation test case, by
+  Jablonowski & Williamson Baroclinic test case Perturbation. JRMS2006
+  and additional computations depicted in DCMIP2016 Test Case Documentation
+  JRMS2006 equations 3, 8, 9, 12, 13 are not computed here
+"""
+# maximum windspeed amplitude - close to windspeed of zonal-mean time-mean
+# jet stream in troposphere
+u0 = 35.0  # From Table VI of DCMIP2016
+# [lon, lat] of zonal wind perturbation centerpoint at 20E, 40N
+pcen = [math.pi / 9.0, 2.0 * math.pi / 9.0]  # From Table VI of DCMIP2016
+u1 = 1.0
+pt0 = 0.0
+eta_0 = 0.252
+eta_s = 1.0  # surface level
+eta_t = 0.2  # tropopause
+t_0 = 288.0
+delta_t = 480000.0
+lapse_rate = 0.005  # From Table VI of DCMIP2016
+surface_pressure = 1.0e5  # units of (Pa), from Table VI of DCMIP2016
+# NOTE RADIUS = 6.3712e6 in FV3 vs Jabowski paper 6.371229e6
+R = constants.RADIUS / 10.0  # Perturbation radiusfor test case 13
```

Comment:
Good comments! Here and throughout the PR.

Reply:
thanks! paid close attention to the paper on this one, which thankfully matches pretty well to what the fortran code does!